Tests/ProcedureKitTests/ConditionTests.swift

```diff
@@ -5,11 +5,80 @@
 //
 
 import XCTest
-import TestingProcedureKit
+@testable import TestingProcedureKit
```

Comment:
Hmm, this might be a problem. I can't remember exactly why this causes issues, but we really only want to testable import ProcedureKit itself. Can we make APIs public here instead?